src/main/java/racingcar/model/Cars.java

```diff
@@ -0,0 +1,37 @@
+package racingcar.model;
+
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+public class Cars {
+    private final List<Car> carList;
+
+    public Cars(String carStr) {
+        this.carList = Arrays.stream(carStr.split(",")).map(Car::new)
+                .collect(Collectors.toList());
```

Comment:
```suggestion
        this.carList = Arrays.stream(carStr.split(","))
                .map(Car::new)
                .collect(Collectors.toList());
```
주요 stream 체이닝에 개행을 해주면 가독성이 더 올라갈 것 같습니다.

Reply:
넵, "stream 체이닝에 개행하기"
사실, 이번 2차 과제를 하면서 stream API를 처음 사용해봐서 어디서 개행을 해야할지 헷갈렸습니다..!